src/clients/python/experimental_api_v2/examples/simple_grpc_v2_class_client.py

```diff
@@ -32,6 +32,7 @@
 from builtins import range
```

Comment:
Should just delete these two class clients... they seem to be just be the same as or a subset of the v2_image_client.

Reply:
These two class clients verify the results of the classification are indeed correct. The v2_image_client.py simply runs the inference but does not verify the results.

Reply:
But you can/should have a CI that runs v2_image_client and verifies it's result. When we shift from V1->V2 we have several existing image_client using CI tests that will provide this coverage.

Reply:
I will make this change in the image_client PR. Including dropping the simple_grpc/http_v2_class_client.py